roles/matrix-synapse/tasks/ext/appservice-discord/setup.yml

```diff
@@ -0,0 +1,73 @@
+---
+- name: Ensure Appservice Discord image is pulled
+  docker_image:
+        name: "{{ matrix_appservice_discord_docker_image }}"
+  when: "matrix_appservice_discord_enabled"
+
+- name: Ensure Appservice Discord configuration path exists
+  file:
+    path: "{{ matrix_appservice_discord_base_path }}"
+    state: directory
+    mode: 0750
+    owner: "{{ matrix_user_username }}"
+    group: "{{ matrix_user_username }}"
+  when: "matrix_appservice_discord_enabled"
+
+- name: Ensure Matrix Appservice Discord config installed
+  copy:
+    content: "{{ matrix_appservice_discord_configuration|to_nice_yaml }}"
+    dest: "{{ matrix_appservice_discord_base_path }}/config.yaml"
+    mode: 0644
+    owner: "{{ matrix_user_username }}"
+    group: "{{ matrix_user_username }}"
+  when: "matrix_appservice_discord_enabled"
+
+- name: Ensure matrix-appservice-discord.service installed
+  template:
+    src: "{{ role_path }}/templates/ext/appservice-discord/systemd/matrix-appservice-discord.service.j2"
+    dest: "/etc/systemd/system/matrix-appservice-discord.service"
+    mode: 0644
```

Comment:
Adding `register: matrix_appservice_discord_systemd_service_result` here will be helpful later, for making systemd automatically get reloaded when the `.service` file changes (see other review comment below).

Note: `result:` should be indented to the same level as `template:`, not nested within it.

Reply:
added this together with the addition of the systemd reload below.
https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/105/commits/278484656b75b70e3cac70fc432ffc6b3672ce7f